tests/bin/test_busd_smoke.py

```
@@ -1,71 +1,71 @@
+        os.environ['TMUX_SESSION'] = 'test-minimal'
+        os.environ['CLAUDE_CMD'] = 'echo "Mock Claude"'
+        
+        # 必要なディレクトリを作成
+        for d in ['mbox/bus/in', 'mbox/pmai/in', 'logs/raw', 'state', 'work']:
+            Path(test_dir, d).mkdir(parents=True, exist_ok=True)
+        
+        # busdをインポート
+        try:
+            import busd
+            print("✓ busd imported successfully")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)